src/connection/server_support.c

```diff
@@ -1819,21 +1824,25 @@ css_pack_message_to_master (const char *server_name)
   size_t message_size =
     strlen (server_name) + 1 + strlen (rel_major_release_string ()) + 1 + strlen (env_name) + 1
     + strlen (pid_string) + 1;
-  /* in order to prepend '#' or server type */
+
+  /* in order to prepend a server type (1 byte char)*/
```

Comment:
How about comment the payload format on together? like

| server_type (1-byte) | #db_name (1-byte + strlen(db_name) | ...

And remove "In order to prepend ..." comments.

Reply:
OK, it will be more clear